app/src/main/java/com/cdio/solitaire/controller/StrategyController.kt

```diff
@@ -55,7 +52,34 @@ class StrategyController {
     fun moveStack() {
         TODO("Analyze code from other repo for data manipulation/evaluation.")
     }
-
+    fun checkFoundationPlusTwoRule(cardToCheck: Card):Boolean{
```

Comment:
Some of these functions could maybe use some documentation, so we can see what their purpose is.